src/components/_modal/pages/bookmarkOrder/multi-order/checkoutForm/MultiOrderCheckoutForm.js

```diff
@@ -46,6 +40,24 @@ const CheckoutForm = ({
   const { pickupBranch, pickupBranchUser, isLoadingBranches } =
     pickupBranchInfo;
 
+  useEffect(() => {
+    setDisabled(
+      !isAnalyzed ||
+        materialsMissingAction > 0 ||
+        materialsNotAllowed > 0 ||
+        duplicateBookmarkIds?.length > 0 ||
+        mail?.valid?.status === false ||
+        numberMaterialsToOrder < 1
+    );
+  }, [
+    isAnalyzed,
+    materialsMissingAction?.length,
+    materialsNotAllowed?.length,
+    duplicateBookmarkIds?.length,
+    mail?.valid?.status,
+    numberMaterialsToOrder,
+  ]);
```

Comment:
Der er vist nogen problemer her:
- `materialMissingAction > 0` giver nok ikke mening, hvis det er en liste, som det ser ud til at være i dependency arrayet (`materialsMissingAction?.length`)
- Samme for `materialsNotAllowed` (men `duplicateBookmarkIds` ser ud til at have length på i `setDisabled()`).
- `numberMaterialsToOrder` ser rigtig ud

Reply:
Godt du så det, materialsMissingAction og materialsNotAllowed er faktisk numbers, ikke arrays. har fjernet length af dem.

Reply:
kalder dem for materialsMissingActionCount, materialsNotAllowedCount og materialsToOrderCount, så bliver jeg ikke forvirret næste gang :-)